packages/data-stores/src/plans/reducer.ts

```diff
@@ -22,12 +26,33 @@ export type PricesMap = {
 	[ slug in PlanSlug ]: string;
 };
 
+export type DiscountsMap = {
+	[ slug in PlanSlug ]: number;
+} & { maxDiscount: number };
+
```

Comment:
Should `PricesMap` and `DiscountsMap` be moved to `packages/data-stores/src/plans/types.ts`?